src/main/java/picard/sam/FilterSamReads.java

```diff
@@ -289,6 +289,9 @@ protected int doWork() {
             final SamReader samReader = SamReaderFactory.makeDefault().referenceSequence(REFERENCE_SEQUENCE).open(INPUT);
             final FilteringSamIterator filteringIterator;
 
+            // Used for exclude/include tag filter
+            List<Object> tagList = TAG_VALUE.stream().map(T -> (Object) T).collect(Collectors.toList());
```

Comment:
didn't you say you're looking for strings? what's this messing around with Objects about?

Reply:
( I realize I asked for it to be refactored...but now, I don't understand why it's needed....)

Reply:
Because if I pass the TagFilter() `TAG_VALUE` which is a `List<String>` it will use this constructor https://github.com/samtools/htsjdk/blob/master/src/main/java/htsjdk/samtools/filter/TagFilter.java#L69 instead of the one it would choose if I pass it a `List<Object>` https://github.com/samtools/htsjdk/blob/master/src/main/java/htsjdk/samtools/filter/TagFilter.java#L80 which makes all the `filterOut` functionality not behave correctly.  There is probably a better/easier way to do this conversion